lib/driver.js

```diff
@@ -476,7 +476,14 @@ class XCUITestDriver extends BaseDriver {
 
     if (this.isSafari() || this.opts.autoWebview) {
       log.debug('Waiting for initial webview');
-      await this.navToInitialWebview();
+      try {
+        await this.navToInitialWebview();
+      } catch (e) {
+        if (!this.isRealDevice()) { throw e; }
+
+        throw Error(`${e.message} Please ensure 'Web Inspector' is enabled on the device, ` +
```

Comment:
We could simply add the tutorial link instead. In general, there might be multiple reasons to why no web view is not found and it makes sense to have a separate troubleshooting tutorial for that, which covers emulators and real device cases

Reply:
Yea, I see it > multiple reasons

Wondered to append a major failure case as `browserName: safari` on a real device here since i got frequent questions about this setting on real devices rather than simulators.

Let me keep the url.

Reply:
I get it. Although, the URL might become confusing for cases where debugging is already enabled, but web views are not visible due to some other reason

Reply:
yea, agree with it > confusing ...